setup.py

```diff
@@ -205,10 +101,21 @@ def check_import(pkgname, pkgver):
 Operating System :: Microsoft :: Windows
 Operating System :: Unix
 """
-setup(name="bcolz",
-      version=VERSION,
-      description='columnar and compressed data containers.',
-      long_description="""\
+
+tests_require = ['mock', 'tox']
+
+if v < (3,):
+    tests_require.append('unittest2')
+    
+setup(
+    name="bcolz",
+    use_scm_version={
```

Comment:
https://pypi.python.org/pypi/setuptools_scm/1.7.0 for reference. It uses the tag as the version number and creates a file with the version number. This eliminates a LOT of boiler plate code in the setup.py , conf.py etc.